os_vif/__init__.py

```diff
@@ -26,10 +26,9 @@
 
 
 def initialize(reset=False, **config):
-    """
-    Loads all os_vif plugins and initializes them with a dictionary of
-    configuration options. These configuration options are passed as-is
-    to the individual VIF plugins that are loaded via stevedore.
+    """Loads all os_vif plugins and initializes them with a dictionary of
```

Comment:
@sahid You should read the above link. :) It specifically says "The summary line may be on the same line as the opening quotes or on the next line."

The forcing of the summary line to be on the same line as the opening quotes is the *silly* thing.